bot/university/university.py

```diff
@@ -129,6 +243,63 @@ def check_reaction(_reaction, user):
         return list_selection_emojis.index(reaction[0].emoji)
 
 
+async def _module_role_error(ctx: commands.Context, error: commands.BadArgument):
+    """Method which posts an error message that the role which should be added/removed from module roles doesn't exist.
+
+    Will always be called by the local error handlers from the subcommands `module add` and `module remove` if the
+    specified role doesn't exist. The error message will be posted in the channel where the command has been invoked
+    and will be deleted shortly after.
+
+    Args:
+        ctx (discord.ext.commands.Context): The context in which the command was called.
+        error (commands.BadArgument): The specific error being raised if the specified role doesn't exist.
+    """
+    role = re.search(r"\"(.*)\"", error.args[0])  # Regex to get the text between two quotes.
```

Comment:
Is it always sure that a BadArgument Error here only has this reason, so only the group can be in quotes?

Reply:
Yes, because the only reason for a Command Error to occur is, if the conversion to a course role doesn't work.